elf_binary.cc

```diff
@@ -181,7 +181,14 @@ std::pair<std::string, std::string> ELFBinary::GetVersion(int index) {
                     if (vna->vna_other == versym_[index]) {
                         LOG(INFO) << "Find Elf_Vernaux corresponds to " << versym_[index] << SOLD_LOG_KEY(strtab_ + vn->vn_file)
                                   << SOLD_LOG_KEY(strtab_ + vna->vna_name);
-                        return std::make_pair(std::string(strtab_ + vn->vn_file), std::string(strtab_ + vna->vna_name));
+
+                        std::string filename = std::string(strtab_ + vn->vn_file);
```

Comment:
Please do not call `map::find` twice. `auto found = filename_to_soname.find(filename);`?

Reply:
I see. I'll fix it.